internal/app/public.go

```diff
@@ -39,6 +39,34 @@ func publicAPI(
 		return c.JSON(http.StatusOK, output)
 	})
 
+	r.POST("/password-reset-request", func(c echo.Context) error {
+		var inp http1.PasswordResetRequestInput
+		if err := c.Bind(&inp); err != nil {
+			return &echo.HTTPError{Code: http.StatusBadRequest, Message: fmt.Errorf("failed to parse request body: %w", err)}
+		}
+
+		output, err := controller.PasswordResetRequest(c.Request().Context(), inp)
+		if err != nil {
+			return err
+		}
+
+		return c.JSON(http.StatusOK, output)
+	})
+
+	r.POST("/password-reset-confirm", func(c echo.Context) error {
```

Comment:
```suggestion
	r.POST("/password-reset", func(c echo.Context) error {
```

Also you can consider combining 2 small APIs into `POST /password` as following:

1. If body contains email, send password reset mail
2. If body contains token and password, reset password
3. Else, return error